test/unit/model/compile/QtiItemAssetCompilerTest.php

```diff
@@ -56,14 +58,21 @@ class QtiItemAssetCompilerTest extends TestCase
     /** @var Directory */
     private $directory;
 
+    /**
+     * @var QtiItemCompilerAssetBlacklist
+     */
+    private $qtiItemNonReplacer;
```

Comment:
Should we rename this attribute?
and is `QtiItemCompilerAssetBlacklist` the good type here?

Reply:
fixed